python3.8/rockcraft.yaml

```diff
@@ -0,0 +1,39 @@
+name: python
+version: '3.8'
+summary: Python 3.8
+description: |
+    Python is an interpreted, interactive, object-oriented, open-source
+    programming language.
+license: GPL-3.0
+
+base: bare
+build-base: ubuntu@20.04
+
+run-user: _daemon_
+
+platforms:
+    amd64:
+
+services:
+    python3:
+        startup: disabled
+        override: replace
+        command: python3
+        on-success: ignore
+        on-failure: ignore
+
+parts:
+    python3:
+        plugin: nil
+        stage-packages:
+            - base-files_base
+            - base-files_release-info
+            - libgcc-s1_libs
+            - libc6_libs
+            - openssl_config
```

Comment:
Please submit the chisel-releases fix we talked about, for the missing symlink target

Reply:
Done: https://github.com/canonical/chisel-releases/pull/100